### DISCUSSION TITLE
Location field types usually confusing

### DISCUSSION THREAD

**Discussion by norkans7** [nyaruka/rapidpro#4418] [Ideas]
We use STATE, DISTRICT and WARD for the location field types but that is not very intuitive in every country

I wonder whether "LOCATION LEVEL 0" (country), "LOCATION LEVEL 1" (state)... would help more?
And be more easier for admins to understand?

**Comment by fredygamez** [comment]
Hi Norbert. Thank you for your help with the location field types. From my experience with Textit Flows and geographical data requests, I have feedback and suggestions.

In response to your suggestion, I agree that using "LOCATION LEVEL 0" (country), "LOCATION LEVEL 1" (state) and so on, could be more intuitive and easier for admins to understand. Additionally, I recommend adding ISO 3166 codes and the admin_levels concept from OpenStreetMap to the location field. This will provide more detailed information and help ensure consistency in location data across different countries.

To provide more information on the ISO 3166 codes, ISO 3166-1 is a standard that defines codes for the names of countries, dependent territories, and special areas of geographical interest. It consists of two-letter country codes, three-letter country codes, and numeric country codes.

ISO 3166-2, on the other hand, defines codes for the names of the principal subdivisions (e.g. states, provinces, regions, etc.) of all countries coded in ISO 3166-1. These codes are composed of a two-letter country code, followed by a hyphen, and then a maximum of three characters representing the subdivision.

In addition, OpenStreetMap (OSM) uses the concept of admin_levels to classify administrative subdivisions. Admin_levels are used to indicate the level of hierarchy of administrative boundaries, and different levels may have different meanings depending on the country or region. For example, admin_level=2 may refer to states or provinces in one country, while in another it may refer to regions or departments.

By incorporating ISO 3166-1 and ISO 3166-2 codes and the admin_levels concept from OpenStreetMap into Textit location field types, you can provide more detailed and consistent information on location data across different countries.

I hope this information is helpful. Let me know if you have any questions or feedback on my end.
